biosimulations/apps/dispatch-service/src/app/app.module.ts

```diff
@@ -63,4 +64,4 @@ import { SubmissionController } from './submission.controller';
     },
   ],
 })
-export class AppModule {}
+export class AppModule { }
```

Comment:
Not specific to this commit, but could you check that you are using prettier as your config, with the .prettier.rc file in the root of the repo? 
We keep oscillating on this. I''ll check my config too.

Reply:
Okay. So I just installed prettier and added below to the vscode settings
```
{
  .....
  .....

  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "[javascript]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode",
    "editor.formatOnSave": true
       }
  }
```